terraform/cp4ba/variables.tf

```diff
@@ -113,37 +110,103 @@ locals {
 
 # --------- DB2 SETTINGS ----------
 variable "enable_db2" {
-  default = false
-  description = "Set to 'true' if you want to install Db2 for your Cloud Pak for Business Automation. If leave 'false', it will not install Db2."
+  default     = true
+  description = "If set to true, it will install DB2 on the given cluster"
 }
 
-variable "db2_project_name" {
-   default = "ibm-db2"
+ variable "db2_project_name" {
+   default     = "ibm-db2"
    description = "The namespace/project for Db2"
  }
-variable "db2_admin_password" {
-  description = "Db2 admin user password defined in LDAP"
-}
 
 variable "db2_admin_username" {
-  default = "db2inst1"
-  description = "Db2 admin username defined in LDAP"
+  default     = "cpadmin"
+  description = "Admin user name defined in LDAP"
 }
 
-variable "db2_host_name" {
-  description = "Host name of Db2 instance"
+variable "db2_user" {
+  default     = "db2inst1"
+  description = "User name defined in LDAP"
 }
 
-variable "db2_host_ip" {
-  description = "IP address for the Db2"
+variable "db2_admin_user_password" {
+  description = "Db2 admin user password defined in LDAP"
 }
 
-variable "db2_port_number" {
-  description = "Port for Db2 instance"
+variable "db2_name" {
+  description = "A name you would like to attribute to your Database. i.e: sample-db2"
+  default     = "sample-db2"
 }
 
 variable "db2_standard_license_key" {
-  description = "The standard license key for the Db2 database product"
+  default     = ""
+  description = "The standard license key for the Db2 database product. Note: the license key is required only for Advanced DB2 installation. Click here to download it: (https://w3-03.ibm.com/software/xl/download/ticket.wss) or [IBM Passport Advantage](https://www.ibm.com/software/passportadvantage/)"
+}
+
+variable "db2_operator_version" {
+  default     = "db2u-operator.v1.1.11"
+  description = "Operator version"
+}
+
+variable "db2_operator_channel" {
+  default     = "v1.1"
+  description = "The Operator Channel performs rollout update when new release is available."
+}
+
+variable "db2_instance_version" {
+  default     = "11.5.6.0"
+  description = "DB2 version to be installed"
+}
+
+variable "db2_cpu" {
+  default     = "4"
+  description = "CPU setting for the pod requests and limits"
+}
+
+variable "db2_memory" {
+  default     = "16Gi"
+  description = "Memory setting for the pod requests and limits"
+}
+
+variable "db2_storage_size" {
+  default     = "150Gi"
+  description = "Storage size for the db2 databases"
+}
+
+variable "db2_storage_class" {
+  default     = "ibmc-file-gold-gid"
+  description = "Name for the Storage Class"
+}
+
+locals {
```

Comment:
Why are these locals?  Wouldn't the user need to define them if they already have a DB2 instance running and don't want to enable DB2 TF module?